src/chk/chk.pb-c.c

```diff
@@ -0,0 +1,559 @@
+/* Generated by the protocol buffer compiler.  DO NOT EDIT! */
+/* Generated from: chk.proto */
+
+/* Do not generate deprecated warnings for self */
+#ifndef PROTOBUF_C__NO_DEPRECATED
+#define PROTOBUF_C__NO_DEPRECATED
+#endif
+
+#include "chk.pb-c.h"
+void   chk__check_report__init
+                     (Chk__CheckReport         *message)
+{
+  static const Chk__CheckReport init_value = CHK__CHECK_REPORT__INIT;
+  *message = init_value;
+}
+size_t chk__check_report__get_packed_size
+                     (const Chk__CheckReport *message)
+{
+  assert(message->base.descriptor == &chk__check_report__descriptor);
+  return protobuf_c_message_get_packed_size ((const ProtobufCMessage*)(message));
+}
+size_t chk__check_report__pack
+                     (const Chk__CheckReport *message,
+                      uint8_t       *out)
+{
+  assert(message->base.descriptor == &chk__check_report__descriptor);
+  return protobuf_c_message_pack ((const ProtobufCMessage*)message, out);
+}
+size_t chk__check_report__pack_to_buffer
+                     (const Chk__CheckReport *message,
+                      ProtobufCBuffer *buffer)
+{
+  assert(message->base.descriptor == &chk__check_report__descriptor);
+  return protobuf_c_message_pack_to_buffer ((const ProtobufCMessage*)message, buffer);
+}
+Chk__CheckReport *
+       chk__check_report__unpack
+                     (ProtobufCAllocator  *allocator,
+                      size_t               len,
+                      const uint8_t       *data)
+{
+  return (Chk__CheckReport *)
+     protobuf_c_message_unpack (&chk__check_report__descriptor,
+                                allocator, len, data);
+}
+void   chk__check_report__free_unpacked
+                     (Chk__CheckReport *message,
+                      ProtobufCAllocator *allocator)
+{
+  if(!message)
+    return;
+  assert(message->base.descriptor == &chk__check_report__descriptor);
+  protobuf_c_message_free_unpacked ((ProtobufCMessage*)message, allocator);
```

Comment:
(style)  "(foo*)" should be "(foo *)"
(style)  space prohibited between function name and open parenthesis '('